tests/bug-detectors/CommandInjection/fuzz.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2023 Code Intelligence GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const child_process = require("child_process");
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const assert = require("assert");
+
+const evilCommand =
+	process.platform === "win32" ? "copy NUL EVIL" : "touch EVIL";
+const safeCommand =
+	process.platform === "win32" ? "copy NUL SAFE" : "touch SAFE";
+
+module.exports.CommandInjectionCallOriginalEvilAsync = async function (
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	/** @type {Buffer} */ unusedData
+) {
+	child_process.execSync(evilCommand);
+};
+
+module.exports.CommandInjectionCallOriginalEvilDoneCallback =
```

Comment:
This is a workaround to make the test pass.
Initially this test was identical to `CommandInjectionCallOriginalEvilAsync` but it was failing.
The first 2 calls to a fuzz function with `done` callback somehow ignore a detected bug.
This needs a bugfix, before we can release our first bug detector.

Reply:
`it("Expect a call to the original exec creating the EVIL file; Done callback", () => {` still seems to be flaky`. It just passed in my latest commit but last night it didnt :(

Reply:
Done callback works properly now :)